dhis-2/dhis-api/src/main/java/org/hisp/dhis/organisationunit/OrganisationUnit.java

```diff
@@ -430,12 +431,13 @@ public boolean isDescendant( Set<OrganisationUnit> ancestors )
         {
             return false;
         }
+        List<String> ancestorsUid = ancestors.stream().map( OrganisationUnit::getUid ).collect( Collectors.toList() );
```

Comment:
Is this done for performance?

Would it be faster to use Set instead of List for faster get lookups?

Reply:
True, fixed